src/pages/howManyProducts.tsx

```diff
@@ -66,11 +69,22 @@ export const isNumberOfProductsAttribute = (
 export const getServerSideProps = (ctx: NextPageContextWithSession): {} => {
     const { fareType } = getSessionAttribute(ctx.req, FARE_TYPE_ATTRIBUTE) as FareType;
     const numberOfProductsAttribute = getSessionAttribute(ctx.req, NUMBER_OF_PRODUCTS_ATTRIBUTE);
+    const ticketType = (getSessionAttribute(ctx.req, TICKET_REPRESENTATION_ATTRIBUTE) as TicketRepresentationAttribute)
+        .name;
+
+    let pageHeading = `How many period tickets do you have for the ${
```

Comment:
in ticket the geoZone message is 

How many period tickets do you have for this geographic zone? 

we currently have 'the'